travis-build/base/README.md

```diff
@@ -0,0 +1,15 @@
+# teracy/travis-build Docker image
```

Comment:
move this file to travis/README.md instead of travis/base/README.md (see other directories to make it consistent)